pylxd/models/certificate.py

```diff
@@ -73,9 +74,9 @@ def create(
             "projects": projects,
         }
         if client.has_api_extension("explicit_trust_token"):
-            data["trust_token"] = secret
+            data["trust_token"] = secret or password
```

Comment:
I feel like we are potentially breaking users anyway unless we dismiss the usage of positional parameters. Could you please remind me whether `pip install pylxd` grabs the latest release or uses the latest changes?

Either way my main point was allowing for password authentication even with the extension present, as I misinterpreted it as meaning that the password auth was deprecated but that is not necessarily the case, it is possible to have the extension and still use password auth. Using what I suggested but keeping the `secret` argument name as is should be better then.

If I am not doing a good job explaining I can hop on a quick call anytime :D

Reply:
`pip install pylxd` pulls the latest release on pypi.

Reply:
Password auth is AFAIK deprecated. Sure you can still use it on 5.21 but that's not recommended and in general when tokens are supported, password should be avoided.

Reply:
Still, don't you think denying password usage in the client even if the server allows it is not ideal?

So considering your point, why not leave `password` as an optional parameter to "force" password usage?